static/app/components/feedback/decodeMailbox.ts

```diff
@@ -9,7 +9,7 @@ export default function decodeMailbox(
     case 'resolved':
       return 'resolved';
     case 'archived':
-      return 'archived';
+      return 'resolved';
```

Comment:
Is the logic... 
If mailbox = anything that isn't `unresolved` --> `resolved`? Or
If mailbox = anything that isn't `resolved` --> `unresolved`?